### PR TITLE
Simplify E57 build and fix warnings.

### PR DESCRIPTION
**Pull request by abellgithub**

### CODE REVIEW

File: plugins/e57/libE57Format/CMakeLists.txt [hunk 1]
```diff
@@ -30,88 +30,20 @@
 # If you find any errors or have suggestion to improve the build script:
 # patches are most welcome! Please send them to the development mailing list.
 
-
-cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
```
Comment (hobu) [inline plugins/e57/libE57Format/CMakeLists.txt]: Why was this deleted? It is required if you want to build the e57 stuff stand-alone?

Comment (abellgithub) [reply plugins/e57/libE57Format/CMakeLists.txt]: This isn't for our E57 stages.  It's for the E57 support library.
